cancellation/src/main.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import { FileBasedCancellationStrategy } from './fileBasedCancellation';
+export { FileBasedCancellationStrategy };
```

Comment:
Fold fileBasedCancellation into main for now.

Reply:
I need some function to be shared in test. so prefer making util files for those functions? I made that way first, but refactored this way since those helper function only useful file based cancellation but not any other implementation.

for example, I was thinking if we go to nexus, I don't think we will suddenly change our implementation to async, meaning we need cross network but still file based cancellation.

anyway, sure I can do that way.